packages/host/app/resources/room.ts

```diff
@@ -227,6 +231,8 @@ export class RoomResource extends Resource<Args> {
         if (!this._fragmentCache.has(event_id)) {
           this._fragmentCache.set(event_id, event.content);
         }
+      } else if (event.content.msgtype === 'org.boxel.commandResult') {
```

Comment:
just a general note, this `if` statement has definitely gotten too large. we should probably employ a strategy pattern here for the various types of matrix events. not necessary for this PR, but maybe you could drop a "TODO" in here so that the next person that wants to add a new condition to this if/else can go down that path.